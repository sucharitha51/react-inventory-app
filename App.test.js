import { configure, shallow } from 'enzyme'
import React from 'react'

import Adapter from 'enzyme-adapter-react-16'
import App from './src/App';
configure({ adapter: new Adapter() })

let wrapper

beforeEach(() => {
  wrapper = shallow(<App />)
})

describe('App  component', () => {

  it('App renders without crashing', () => {
    expect(wrapper.exists()).toEqual(true);
  });

  it('App should match with snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
})