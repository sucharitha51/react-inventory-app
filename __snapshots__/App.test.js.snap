// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App  component App should match with snapshot 1`] = `"<header><h1>My Inventory</h1><div class=\\"actions\\"><span class=\\"add-new\\">Add New Item</span><button style=\\"cursor:pointer;border:1px solid #ccc;border-radius:15px;outline:none;padding:5px 10px;font-size:14px;margin-right:10px;background-color:rgb(204, 204, 204)\\">Export</button></div></header><div class=\\"items-list\\"><div class=\\"columns\\"><p class=\\"id\\">#</p><p class=\\"name\\">Name</p><p class=\\"serialNumber\\">Serial Number</p><p class=\\"price\\">Value</p></div><p>No data.</p></div>"`;
